### PR TITLE
Update status automatically after checking job status

### PR DESCRIPTION
**Pull request by wewang58**
@rioliu Update task status automatically for stage-testing and image-consistency-check.
1.$ oar -r 4.13.6 stage-testing -n 783
2023-07-26T09:33:25Z: INFO: check stage job status according to job id:783
2023-07-26T09:33:31Z: INFO: Stage-Pipeline job: 783 status is: SUCCESS
2023-07-26T09:33:31Z: INFO: SUCCESS
2023-07-26T09:33:32Z: INFO: task [Stage testing] status is changed to [Pass]
2023-07-26T09:33:42Z: INFO: Overall status is updated to Green

2. $oar -r 4.13.6 stage-testing -n 787
2023-07-26T09:29:29Z: INFO: check stage job status according to job id:787
2023-07-26T09:29:34Z: INFO: Stage-Pipeline job: 787 status is: ABORTED  # also  failed to set Fail
2023-07-26T09:29:36Z: INFO: task [Stage testing] status is changed to [Fail]
2023-07-26T09:29:38Z: INFO: Overall status is updated to Red

3. $ oar -r 4.13.6 stage-testing -n 788
2023-07-26T09:34:48Z: INFO: check stage job status according to job id:788
2023-07-26T09:34:52Z: INFO: Stage-Pipeline job: 788 status is: In Progress
2023-07-26T09:34:52Z: INFO: In Progress
2023-07-26T09:34:53Z: INFO: task [Stage testing] status is changed to [In Progress]

### CODE REVIEW

File: oar/cli/cmd_stage_testing.py [hunk 1]
```diff
@@ -36,12 +35,20 @@ def stage_testing(ctx, build_number):
...
                     report.update_task_status(LABEL_TASK_STAGE_TEST, TASK_STATUS_INPROGRESS)
                 else:
                     logger.info("push to cdn stage is not completed, so will not trigger stage test")
         except Exception as we:
             logger.exception("trigger stage testing failed")
     else: 
         logger.info(f"check stage job status according to job id:{build_number}")
-        jh.get_job_status(jh.zstream_url, "Stage-Pipeline", build_number)
+        job_status= jh.get_job_status(jh.zstream_url, "Stage-Pipeline", build_number)
+        logger.info(f"{job_status}")
```
Comment (rioliu-rh) [inline oar/cli/cmd_stage_testing.py:47]:
```suggestion
        logger.info(job_status)
```

Comment (wewang58) [reply oar/cli/cmd_stage_testing.py]: @rioliu-rh  I deleted this, it's debug info

**Pull request merged**